druid/druid-api/src/services/HttpService.ts

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import * as requestService from "request";
+import { config } from "../configs/config";
+import { APILogger } from "./ApiLogger";
+
+/**
+ * It provides the service to interact with the an external system.
+ */
+export class HttpService {
```

Comment:
I think this class is redundant. Just use a simple "request" module.  Also what advantage does axios give?